### PR TITLE
[stable/nextcloud] deprecate nextcloud helm chart

### PR DESCRIPTION
**Pull request by billimek**
#### What this PR does / why we need it:

This deprecates the nextcloud chart and references the new home.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/nextcloud/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: nextcloud
-version: 1.12.0
+version: 1.12.1
 appVersion: 17.0.0
 description: A file sharing server that puts the control and security of your own data back into your hands.
```
Comment (scottrigby) [inline stable/nextcloud/Chart.yaml]: Also please prefix the description with "DEPRECATED" (see [Deprecating A Chart](https://github.com/helm/charts/blob/master/PROCESSES.md#deprecating-a-chart))

File: stable/nextcloud/README.md [hunk 2]
```diff
@@ -1,4 +1,9 @@
-# nextcloud
+# DEPRECATED - nextcloud
+
+|**This chart has been deprecated and moved to its new home:**
```
Comment (scottrigby) [inline stable/nextcloud/README.md:3]: Also please add to NOTES.txt (see [Deprecating A Chart](https://github.com/helm/charts/blob/master/PROCESSES.md#deprecating-a-chart))

**Review by scottrigby — CHANGES_REQUESTED** [review]
Nice 👍 just a few comments

**Comment by scottrigby** [comment]
Added link to this PR in #21103

File: stable/nextcloud/Chart.yaml (hunk 1 above)
Comment (billimek) [reply stable/nextcloud/Chart.yaml]: done!

File: stable/nextcloud/README.md (hunk 2 above)
Comment (billimek) [reply stable/nextcloud/README.md:3]: ... and done!

**Review by scottrigby — APPROVED** [review]
@billimek looks good! Should we wait to merge this until after https://github.com/nextcloud/helm/issues/7 is finished?

**Comment by billimek** [comment]
> @billimek looks good! Should we wait to merge this until after [nextcloud/helm#7](https://github.com/nextcloud/helm/issues/7) is finished?

That should be completed now, I think we are safe to merge.  Appreciate the help and guidance!

**Pull request merged**